peewee.py

```diff
@@ -2437,15 +2446,31 @@ def filter(self, *args, **kwargs):
 
         dq_node = dq_node.rhs
 
+        seen = set()
+        remove_dupes = lambda lst: seen.clear() or [
+            i for i in lst if i not in seen and not seen.add(i)]
```

Comment:
`set.clear()` and `set.add()` seem to always return `None`. What's going on here?